docs/training/solo.rst

```diff
@@ -0,0 +1,195 @@
+
+================================
+Aloha Solo Training $ Evaluation
+================================
+
+.. note::
+
+  **ACT Solo** is supported exclusively with **Aloha 2.0**.
+  There is no support for **Aloha Solo** on **Aloha 1.0**.
+  Ensure that you are using compatible software versions to avoid any issues during training or evaluation.
+
+
+Install Dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+Install the necessary dependencies inside your containerized environment:
+
+.. code-block:: bash
+
+    $ pip install dm_control==1.0.14
+    $ pip install einops
+    $ pip install h5py
+    $ pip install ipython
+    $ pip install matplotlib
+    $ pip install mujoco==2.3.7
+    $ pip install opencv-python
+    $ pip install packaging
+    $ pip install pexpect
+    $ pip install pyquaternion
+    $ pip install pyyaml
+    $ pip install rospkg
+    $ pip install torch
+    $ pip install torchvision
+
+Clone ACT Solo Repository
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Clone ACT Solo if using Aloha Solo
+
+.. code-block:: bash
+
+    $ cd ~
+    $ git clone https://github.com/Interbotix/act_solo.git act_training_evaluation
```

Comment:
I'm not sure having them clone a branch is the best idea. Do we want to add act_solo to the main act repo? We should at least have them clone the `main` branch of this repo.

Reply:
It's the main branch. It's just the 'act_training_evaluation' is just the name of the directory that it will be loacally stored in.